curiouslearning/FeedTheMonsterJS
# fix - Cr 155 Development build and verify "CV Creole" FTM

# Changes
- caboverdecreole language folder(build and lang folder)

# How to test
- Select the caboverdecreole from drop down to check content

Ref: [CR-155](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-155)


[CR-155]: https://curiouslearning.atlassian.net/browse/CR-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
